### DISCUSSION TITLE
Leantime 2.2.9 released

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#1048] [Announcements]
Quite a few patches in this one. https://github.com/Leantime/leantime/releases/tag/v2.2.9
Please update as soon as possible

**Comment by joaobrazuna** [comment]
Hi,
The most urgent problems are now fixed.
Thanks
The Ldap error when creating a new user continues
The sequence is:
- Login 1st time with a new LDAP user. (works ok)
- Go to "My profile" of the new user  (all data is correct.)
- Do any change and Save. The following error is reported: "Not a valid email address" (all data is now missing)
![image](https://user-images.githubusercontent.com/44297535/196646705-f1db8135-3728-4065-a3a6-4bf19a2767c8.png)

**Comment by marcelfolaron** [comment]
Oh I see, thanks for the report. I found the the issue. Will do another
patch tonight.
